google/cloud/storage/retry.py

```diff
@@ -14,18 +14,18 @@
 
 import requests
 
-from google.api_core import exceptions
+from google.api_core import exceptions as api_exceptions
```

Comment:
No longer has to be aliased since you're not importing the auth exceptions as well.